src/gantt.ts

```diff
@@ -866,22 +930,21 @@ export class Gantt implements IVisual {
             return tasks;
         }
 
-        const colorHelper: ColorHelper = new ColorHelper(colors, Gantt.LegendPropertyIdentifier);
+        const colorHelper: ColorHelper = new ColorHelper(this.colors, Gantt.LegendPropertyIdentifier);
         const groupValues: GanttColumns<DataViewValueColumn>[] = GanttColumns.getGroupedValueColumns(dataView);
-        const sortingOptions: SortingOptions = Gantt.getSortingOptions(dataView);
 
-        const collapsedTasks: string[] = JSON.parse(settings.collapsedTasksCardSettings.list.value);
-        let durationUnit: DurationUnit = <DurationUnit>settings.generalCardSettings.durationUnit.value.value.toString();
-        let duration: number = settings.generalCardSettings.durationMin.value;
+        const collapsedTasks: string[] = JSON.parse(this.formattingSettings.collapsedTasksCardSettings.list.value);
+        let durationUnit: DurationUnit = <DurationUnit>this.formattingSettings.generalCardSettings.durationUnit.value.value.toString();
+        let duration: number = this.formattingSettings.generalCardSettings.durationMin.value;
 
         let endDate: Date = null;
 
         values.Task.forEach((categoryValue: PrimitiveValue, index: number) => {
-            const selectionBuilder: ISelectionIdBuilder = host
+            const selectionBuilder: ISelectionIdBuilder = this.host
                 .createSelectionIdBuilder()
-                .withCategory(dataView.categorical.categories[0], index);
+                .withCategory(dataView.categorical.categories[0], index); // TODO: Add logic finding the actual index instead of depending on the order
```

Comment:
this TODO comment. Do you plan to work on it later in the future or forget it now?